security/login_link.rst

```diff
@@ -0,0 +1,510 @@
+    be used only if the current request is covered by a
+    :ref:`security firewall <security-firewalls>`.
+
+Import Considerations
+---------------------
+
+Login links are a convinient way of authenticating users, but it is also
+considered less secure than a traditional username and password form. It is
+not recommended to use login links in security critical applications.
+
+However, the implementation in Symfony does have a couple extension points
+to make the login links more secure. In this section, the most important
+configuration decisions are discussed:
+
+* `Limit Login Link Lifetime`_
+* `Invalidate Login Links`_
+* `Only allow a Link to be used Once`_
+
+Limit Login Link Lifetime
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is important for login links to have a limited lifetime. This reduces
+the chance that someone can intercept the link and use it to login as
+somebody else. By default, Symfony defines a lifetime of 10 minutes (600
+seconds). You can customize this using the ``lifetime`` option:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/security.yaml
+        security:
+            firewalls:
+                main:
+                    login_link:
+                        check_route: login_check
+                        # lifetime in seconds
+                        lifetime: 300
+
+    .. code-block:: xml
+
+        <!-- config/packages/security.xml -->
+        <?xml version="1.0" encoding="UTF-8"?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:srv="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/security
+                https://symfony.com/schema/dic/security/security-1.0.xsd">
+
+            <config>
+                <firewall name="main">
+                    <!-- lifetime: lifetime in seconds -->
+                    <login-link check-route="login_check"
+                        lifetime="300"
+                    />
+                </firewall>
+            </config>
+        </srv:container>
+
+    .. code-block:: php
+
+        // config/packages/security.php
+        $container->loadFromExtension('security', [
+            'firewalls' => [
+                'main' => [
+                    'login_link' => [
+                        'check_route' => 'login_check',
+                        // lifetime in seconds
+                        'lifetime' => 300,
+                    ],
+                ],
+            ],
+        ]);
+
+Invalidate Login Links
+~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony uses signed URLs to implement login links. The advantage of this is
+that links do not have to be stored in a database in order to invalidate
+them. It is a good idea to invalidate the link whenever important user
```

Comment:
The 2nd sentence seems odd to me. The advantage of signed URLs is that we don't need to store *valid* login links in the database. The disadvantage is actually that we then need to store *invalid* links somewhere.